vaporetto/src/char_scorer.rs

```diff
@@ -9,7 +9,11 @@ use bincode::{
     error::{DecodeError, EncodeError},
     BorrowDecode, Decode, Encode,
 };
+
+#[cfg(not(feature = "charwise-daachorse"))]
 use daachorse::DoubleArrayAhoCorasick;
+#[cfg(feature = "charwise-daachorse")]
+type DoubleArrayAhoCorasick = daachorse::charwise::CharwiseDoubleArrayAhoCorasick;
```

Comment:
Function signatures of CharwiseDAAC and DAAC are different, so I think this renaming confuses.
(e.g., `find_overlapping_no_suffix_iter_from_iter()` is not unsafe in DAAC, but is unsafe in CharwiseDAAC.)

Reply:
I removed the renaming.